modules/testing/exercise.md

```diff
@@ -0,0 +1,11 @@
+---
+title: Testing Exercise
+type: exercise
+order: 4
+---
+
+# Exercise 2
+
+Identify the test types in this story:
+
+During the process of manufacturing a ballpoint pen, we see smoke coming out of one of our machines, so we halt production immediately, put out the fire, and check why it caught fire in the first place occurred before continuing. We produce the cap, the body, the tail, the ink cartridge, and the ballpoint in different machines and check the quality of each component before assembling the whole. Next we check that the cap and tail fit on the body and the ink cartridge fits inside. When everything is put together, we ensure that it writes properly and doesn't leak. In the future, we plan to supply different colors of ink and will re-run these tests for each new colored pen.
```

Comment:
"in the first place <occurred> before continuing". Nice example!

Reply:
it's stolen/adapted from an existing exercise ;)